src/lib/containers/provenance/ExpandGraphNodeLabel.tsx

```diff
@@ -0,0 +1,21 @@
+import pluralize from 'pluralize'
+import React, { useMemo } from 'react'
+import { EntityHeaderIsCurrent } from './ProvenanceUtils'
+
+export type ExpandGraphNodeDataProps = {
+  entityHeaderIsCurrent: EntityHeaderIsCurrent
```

Comment:
Can be removed?

Reply:
No, used in ProvenanceGraph when an Expand node is clicked (on [this line](https://github.com/Sage-Bionetworks/Synapse-React-Client/pull/1695/files#diff-5b8ab3931db415f75971a6eb3cdbad84cbb2176634ea29b856ad3c4454c717ecR623))